app/repositories/CalculationRepository.scala

```diff
@@ -53,56 +49,71 @@ trait CalculationRepository extends ReactiveRepository[CachedCalculation, BSONOb
 }
 
 @Singleton
-class CalculationMongoRepository @Inject()(component: ReactiveMongoComponent)
-  extends ReactiveRepository[CachedCalculation, BSONObjectID](
-    "calculation",
-    component.mongoConnector.db,
-    CachedCalculation.formats) with CalculationRepository {
+class CalculationMongoRepository @Inject()(mongo: MongoComponent)
+  extends PlayMongoRepository[CachedCalculation](
+    collectionName = "calculation",
+    mongoComponent = mongo,
+    domainFormat = CachedCalculation.formats,
+    indexes = Seq.empty
+  ) with CalculationRepository {
 
   val fieldName = "createdAt"
   val createdIndexName = "calculationResponseExpiry"
   val expireAfterSeconds = "expireAfterSeconds"
   val timeToLive = 600
 
+  val logger: Logger = Logger(this.getClass)
```

Comment:
Other classes in this Repo use extends Logging, rather than Logger(this.getClass) .

It would be best to be consistent here with the other classes

Reply:
done https://github.com/hmrc/gmp/pull/60/commits/3e2e3d9e3612627543edefa737ce6533cdce25ac